types/express-http-proxy/index.d.ts

```diff
@@ -12,10 +13,43 @@ import { Response } from "express-serve-static-core";
 
 interface ProxyOptions {
     proxyReqPathResolver?: (req: Request) => string;
-    proxyReqOptDecorator?: (proxyReqOpts: RequestOptions, srcReq: Request) => RequestOptions;
-    userResHeaderDecorator?: (headers: IncomingHttpHeaders, userReq: Request, userRes: Response, proxyReq: Request, proxyRes: Response) => OutgoingHttpHeaders;
+    proxyReqOptDecorator?: (
+        proxyReqOpts: RequestOptions,
+        srcReq: Request
+    ) => RequestOptions;
+    userResHeaderDecorator?: (
+        headers: IncomingHttpHeaders,
+        userReq: Request,
+        userRes: Response,
+        proxyReq: Request,
+        proxyRes: Response
+    ) => OutgoingHttpHeaders;
+    filter?: (req: Request, res: Response) => boolean;
+    userResDecorator?: (
+        proxyRes: Response,
+        proxyResData: JSON,
+        userReq: Request,
+        userRes: Response
+    ) => Promise<string>;
+    limit?: string;
```

Comment:
This type should be `number | string`.

See [here](https://github.com/villadora/express-http-proxy/blob/5bada09cca1567cbb69a6ec3a637f6457f9e9495/lib/requestOptions.js#L96-L98) and [these docs](https://www.npmjs.com/package/raw-body#getrawbodystream-options-callback).